fuzz/Cargo.lock

```diff
@@ -0,0 +1,1051 @@
+# This file is automatically @generated by Cargo.
```

Comment:
I'd recommend to avoid having a second `Cargo.lock` and instead make it a workspace member by adding this to the parent `Cargo.toml`:

```
[workspace]
members = ["fuzz"]
```

This was recently made possible in cargo-fuzz 0.11.4 (https://github.com/rust-fuzz/cargo-fuzz/pull/357) :)

Reply:
Hmm... I wasn't sure if I could have parent project in a normal layout and also fuzz as a workspace member but apparently inserting:

```toml
[workspace]
members = [".", "fuzz"]
```

in the parent Cargo.toml worked :exploding_head: